pkg/util/util_test.go

```diff
@@ -53,6 +53,63 @@ func TestUntilReturnsImmediately(t *testing.T) {
 	}
 }
 
+func TestJitterUntil(t *testing.T) {
+	ch := make(chan struct{})
+	close(ch)
```

Comment:
that's great, add a comment so readers who are unfamiliar with the code get it on 1st pass.